pkg/provider/aws/resources/vpc.go

```diff
@@ -76,7 +76,7 @@ func CreateNetwork(appName string, dag *core.ResourceGraph) {
 	dag.AddResource(region)
 	dag.AddResource(vpc)
 	dag.AddResource(igw)
-	dag.AddDependency(vpc, igw)
+	dag.AddDependency(igw, vpc)
```

Comment:
how are the unit tests for createNetwork still passing also? They check for deps

Reply:
The test was actually ineffectual since `GetDependency` would never return nil. This is updated (to return nil) and the test changed in the recent commit.